components/content-blocks/Callout/CalloutEntry/styles.js

```diff
@@ -0,0 +1,206 @@
+import styled from "styled-components";
+import { MixedLink } from "@rubin-epo/epo-react-lib";
+import SharePopupComponent from "@/layout/SharePopup";
+import {
+  fluidScale,
+  containerRegular,
+  respond,
+  tokens,
+} from "@/styles/globalStyles";
+import { Section as CalloutSection } from "../styles";
+
+export const Section = styled(CalloutSection)`
+  position: relative;
+`;
+
+export const Inner = styled(MixedLink)`
+  ${containerRegular()}
+  position: ${(p) => p.$position};
+  display: grid;
+  gap: 40px;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+    "image title"
+    "image subtitle"
+    "image text"
+    "image footer";
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 10px;
+  grid-column-gap: 30px;
+  min-height: 360px;
+  height: 100%;
+  padding-top: ${fluidScale("50px", "30px")};
+  padding-bottom: ${fluidScale("50px", "40px")};
+  align-content: start;
+  text-decoration: none;
+  transition: color 0.2s, background-color 0.2s;
+
+  // set the grid areas for various bits
+  .image {
```

Comment:
you can do this without setting additional classnames, since `ImageWrapper` is being defined in this styles module you can replace

```
.images { ... }
```

with 

```
${ImageWrapper} {...}
```

and styled-components will extract the generated classname for ImageWrapper and use that